app-emulation/open-vm-tools/open-vm-tools-9.10.0.ebuild

```diff
@@ -17,7 +17,7 @@ EGIT_SOURCEDIR="${WORKDIR}"
 
 LICENSE="LGPL-2"
 SLOT="0"
-KEYWORDS="amd64 ~x86"
+KEYWORDS="amd64 arm64 ~x86"
```

Comment:
open-vm-tools is only applicable to VMware so I doubt this makes sense. Probably should fix by using `arm64? (  app-emulation/open-vm-tools )` or similar in board-packages.